### PR TITLE
chore(Module/Structure): Change structure for new modules

### PR DESCRIPTION
**Pull request by pangolp**
Due to some recent changes in the structure that the modules must have, the current structure must be updated, because when creating a module for the first time, this template is downloaded.

WoWScrnShot_060923_155858

### CODE REVIEW

File: conf/conf.sh.dist [hunk 1]
```diff
@@ -1,32 +0,0 @@
...
-	"$MOD_SKELETON_ROOT/sql/auth/updates/"
-)
-
-DB_CHARACTERS_UPDATES_PATHS+=(
-	"$MOD_SKELETON_ROOT/sql/characters/updates/"
-)
-
-DB_WORLD_UPDATES_PATHS+=(
-	"$MOD_SKELETON_ROOT/sql/world/updates/"
-)
```
Comment (Helias) [inline conf/conf.sh.dist:32]: why you removed this? to use some other default confs?

Comment (pangolp) [reply conf/conf.sh.dist:32]: Apparently yes. I saw it in another module.
https://github.com/azerothcore/mod-weekend-xp
The idea is that it executes the SQL automatically.
In fact it did, I did not run any SQL.

Comment (Helias) [reply conf/conf.sh.dist:32]: ok 👍 , I was thinking that this skeleton had the right directory structure for this, but apparently I was wrong

File: src/MyPlayer.cpp [hunk 2]
```diff
@@ -17,7 +17,7 @@ class MyPlayer : public PlayerScript
     {
         if (sConfigMgr->GetOption<bool>("MyModule.Enable", false))
         {
-            ChatHandler(player->GetSession()).SendSysMessage("Hello World from Skeleton-Module!");
+            ChatHandler(player->GetSession()).PSendSysMessage(35410);
```
Comment (Helias) [inline src/MyPlayer.cpp:20]: I would like to put a better example here replacing the magic number 35410 with an enum and put the enum here, example:
```C++
enum AcoreString {
  HELLO_WORLD = 45410
};
....
ChatHandler(player->GetSession()).PSendSysMessage(HELLO_WORLD);
```

Comment (pangolp) [reply src/MyPlayer.cpp]: Ready

**Review by Helias — APPROVED** [review]

**Comment by pangolp** [comment]
Thanks @Helias

**Comment by pangolp** [comment]
I also added the issue template and the pull request.
This way the new modules have the most updated structure.

**Comment by pangolp** [comment]
I was testing the hook, and I missed a method that does not correspond to this module. So I updated it.

**Pull request merged**